app/boards/shields/kyria/keymap/keymap.overlay

```diff
@@ -0,0 +1,34 @@
+#include <behaviors.dtsi>
+#include <dt-bindings/zmk/keys.h>
+#include <dt-bindings/zmk/matrix-transform.h>
+
+/ {
+	chosen {
+		zmk,keymap = &keymap0;
+	};
+	
+	keymap0: keymap {
+		compatible = "zmk,keymap";
+		label ="Default Kyria Keymap";
+		layers = <&default>;
+	};
+
+	layers {
+		compatible = "zmk,layers";
+
+		default: layer_0 {
+			label = "DEFAULT";
+// ---------------------------------------------------------------------------------------------------------------------------------
+// |  ESC  |  Q  |  W  |  E   |  R   |  T   |                                          |  Y   |  U    |  I    |  O   |   P   | "|" |
+// |  <-   |  A  |  S  |  D   |  F   |  G   |                                          |  H   |  J    |  K    |  L   |   ;   |  '  |
+// | SHIFT |  Z  |  X  |  C   |  V   |  B   | L SHIFT | L SHIFT |  | L SHIFT | L SHIFT |  N   |  M    |  ,    |  .   |   /   |  -  |
+//                     | GUI  | DEL  | RET  |  SPACE  |   ESC   |  |   RET   |  SPACE  | TAB  | BSPC  | R-ALT |
+			bindings = <
+	&kp ESC  &kp Q &kp W &kp E &kp R &kp T                                            &kp Y &kp U  &kp I    &kp O   &kp P    &kp PIPE
```

Comment:
A more realistic keymap after the refactor.

The behaviors are exposed as DT names you can reference. In this PR are:

* `&kp A` - key press
* `&mo 1` - momentary layer.
* `&mt MOD_LSFT A` - mod-tap behavior.
* `&reset` - device reset (doesn't enter DFU yet)
* `&trans` - Transparent, used on layes above `&mo` keys mostly.